unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HotScenesController/HotScenesController.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Collections.Generic;
+using DCL.Helpers;
+using UnityEngine;
+
+[assembly: System.Runtime.CompilerServices.InternalsVisibleTo("HotScenesControllerTests")]
+
+public class HotScenesController : MonoBehaviour
+{
+    public static HotScenesController i { get; private set; }
+
+    public event Action OnHotSceneListFinishUpdating;
+    public event Action OnHotSceneListChunkUpdated;
+
+    public List<HotSceneInfo> hotScenesList { get; private set; } = new List<HotSceneInfo>();
+    public bool isUpdating { get; private set; }
+    public float timeSinceLastUpdate { get { return Time.realtimeSinceStartup - lastUpdateTime; } }
+
+    private float lastUpdateTime = float.MinValue * .5f;
+
+    [Serializable]
+    public struct HotSceneInfo
+    {
+        [Serializable]
+        public struct Realm
+        {
+            public string serverName;
+            public string layer;
+            public int usersCount;
+            public int usersMax;
+        }
+        public Vector2Int baseCoords;
+        public int usersTotalCount;
+        public Realm[] realms;
+    }
+
+    [Serializable]
+    internal struct HotScenesUpdatePayload
+    {
+        public int chunkIndex;
+        public int totalScenes;
+        public HotSceneInfo[] scenesInfo;
+    }
+
+    void Awake()
+    {
+        i = this;
+    }
+
+    public void UpdateHotScenesList(string json)
+    {
+        var updatePayload = Utils.SafeFromJson<HotScenesUpdatePayload>(json);
+
+        if (updatePayload.chunkIndex == 0)
+        {
+            isUpdating = true;
+            hotScenesList.Clear();
+        }
+
+        hotScenesList.AddRange(updatePayload.scenesInfo);
+        OnHotSceneListChunkUpdated?.Invoke();
+
+        if (hotScenesList.Count >= updatePayload.totalScenes)
```

Comment:
if we send "chunksCount" or "lastChunkIndex" (instead of `totalScenes`) we can compare `updatePayload.chunkIndex >= lastChunkIndex` instead of accesing the more costly `hotScenesList.Count`.

But i doubt the current approach would affect performance at all.